### ISSUE TITLE
stopped working after UI update

### ISSUE THREAD

**Issue by catarium** [hnmr293/posex#39]
![chrome_TcW6MmDfnu](https://github.com/hnmr293/posex/assets/60063501/c99e78a4-c76a-442b-aca5-df8099dc8f26)

**Comment by sklbz** [comment]
same

**Comment by Corvus-Malus** [comment]
same.

**Comment by HOMELANDER-1** [comment]
the rollback of "AUTOMATIC 1111 stable-diffusion-webui" `git checkout 72cd27a13587c9579942577e9e3880778be195f6` helped me

**Comment by Corvus-Malus** [comment]
> ![chrome_TcW6MmDfnu](https://user-images.githubusercontent.com/60063501/238127497-c99e78a4-c76a-442b-aca5-df8099dc8f26.gif)


the rollback of "AUTOMATIC 1111 stable-diffusion-webui" `git checkout 72cd27a13587c9579942577e9e3880778be195f6` helped me
https://i.imgur.com/hmocAab.jpeg

**Comment by bekertughan** [comment]
> > ![chrome_TcW6MmDfnu](https://user-images.githubusercontent.com/60063501/238127497-c99e78a4-c76a-442b-aca5-df8099dc8f26.gif)
> >       
> >         [
> >           
> >         
> >             ![chrome_TcW6MmDfnu](https://user-images.githubusercontent.com/60063501/238127497-c99e78a4-c76a-442b-aca5-df8099dc8f26.gif)
> >           ](https://user-images.githubusercontent.com/60063501/238127497-c99e78a4-c76a-442b-aca5-df8099dc8f26.gif)
> >         
> >         
> >           
> >             
> >               
> >             
> >             
> >               
> >               
> >             
> >           
> >           [
> >             
> >               
> >             
> >           ](https://user-images.githubusercontent.com/60063501/238127497-c99e78a4-c76a-442b-aca5-df8099dc8f26.gif)
> 
> `git checkout 72cd27a13587c9579942577e9e3880778be195f6`мне помог откат "AUTOMATIC 1111 stable-diffusion-webui" https://i.imgur.com/hmocAab.jpeg

thanks, your method worked for me

**Comment by catarium** [comment]
> > ![chrome_TcW6MmDfnu](https://user-images.githubusercontent.com/60063501/238127497-c99e78a4-c76a-442b-aca5-df8099dc8f26.gif)
> >       
> >         [
> >           
> >         
> >             ![chrome_TcW6MmDfnu](https://user-images.githubusercontent.com/60063501/238127497-c99e78a4-c76a-442b-aca5-df8099dc8f26.gif)
> >           ](https://user-images.githubusercontent.com/60063501/238127497-c99e78a4-c76a-442b-aca5-df8099dc8f26.gif)
> >         
> >         
> >           
> >             
> >               
> >             
> >             
> >               
> >               
> >             
> >           
> >           [
> >             
> >               
> >             
> >           ](https://user-images.githubusercontent.com/60063501/238127497-c99e78a4-c76a-442b-aca5-df8099dc8f26.gif)
> 
> the rollback of "AUTOMATIC 1111 stable-diffusion-webui" `git checkout 72cd27a13587c9579942577e9e3880778be195f6` helped me https://i.imgur.com/hmocAab.jpeg

this broke my models so I did checkout to other version

**Comment by 23fbhtg34** [comment]
git checkout 72cd27a13587c9579942577e9e3880778be195f6 
yesterday it worked now after updating sd and controlnet just a black picture

**Comment by 1d618** [comment]
I use the latest versions of SD-webui, posex and sd-webui-controlnet, and until yesterday I had the same problems that the editor did not appear after installing the check mark. and by chance I solved this problem...
I install extensions (via extensions -> available -> load from) **sd-webui-ar** and **canvas-zoom**
after that, posex started working.
I don't know how or what this is related to, but now it works. maybe it will help someone too.

**Comment by tjea** [comment]
> I use the latest versions of SD-webui, posex and sd-webui-controlnet, and until yesterday I had the same problems that the editor did not appear after installing the check mark. and by chance I solved this problem... I install extensions (via extensions -> available -> load from) **sd-webui-ar** and **canvas-zoom** after that, posex started working. I don't know how or what this is related to, but now it works. maybe it will help someone too.

Yes, after I installed the `canvas-zoom` the issue is gone.

**Comment by daswer123** [comment]
Guys, as a Canvas Zoom developer I can explain this.

See canvas-zoom uses a modified gradio, which I put together myself from source and replace the main one, so that some of the extension's features would work.

Since webui 1.3 automatic updated the gradio to version 3.31 and I recently updated the modified gradio with it as well. However, as it turned out posex stopped working on version 3.31 and installing canvas-zoom helped since it was replacing gradio with a modified version 3.28.1. 

Installing canvas-zoom from the extension tab won't help you now. 
However, I made a backup of the extension just in case before updating the modified gradio (3.28.1). You can install directly via install from url

the old version canvas-zoom where posex works
https://github.com/daswer123/canvas-zoom

**Comment by daswer123** [comment]
You can also roll back posex to 0.4. You can do this as described [here](https://github.com/hnmr293/posex/issues/40#issuecomment-1574551072) or you can temporarily download the version I already rolled back [https://github.com/daswer123/posex](https://github.com/daswer123/posex) via install from url

This will not limit you to the gradio version or the extension version.

**Comment by Eloren1** [comment]
@daswer123 Thanks. But why hasn't anyone made an official solution a few months later? Why doesn't everyone care?

**Comment by RoyalGoose** [comment]
Is there any updates?

**Comment by sonicpeter** [comment]
Is there any updates?

**Comment by goldmojo** [comment]
> 

The current 0.4.0 rollback is working fine.
What do you expect from a newer version?

**Comment by lokimckay** [comment]
If only 0.4 works in the current A1111 environment, wouldn't it make sense to roll back to 0.4 until a fix that allows the newer features to work with the latest gradio is implemented?

**Comment by goldmojo** [comment]
Agree, you can install the rolled back from @daswer123 URL https://github.com/daswer123/posex

**Comment by lokimckay** [comment]
That repo is not the one linked to by the A1111 UI

The point is to avoid everyone having to google the issue, so IMO either A1111 should be updated to point to that repo, or this one should be rolled back to a stable commit

(All of this is assuming that A1111 is the primary UI that the dev wants to target)

**Comment by sonicpeter** [comment]
It is strange to roll back to an old version of posex though...

**Comment by lokimckay** [comment]
Not strange at all, many repositories use a main / develop branch system or similar where the main branch is stable - and doesnt include the latest features that are being worked on

**Comment by onlineapps-cloud** [comment]
same problem

**Comment by Patrick-81** [comment]
Same problem here after the last update of A1111.

**Comment by mqopi** [comment]
same

**Comment by wensenz** [comment]
same

**Comment by witchpiggie** [comment]
This is still a problem. Neither rolling back to 0.4 nor installing @daswer123 's version works. All I get is the checkbox 'send this image to controlnet'. Seems like a proper fix is in order.

**Comment by nothingness6** [comment]
I faced the same issue. Tested in 1.8.0 and 1.7.0, but no luck. 
No solution yet?

**Comment by neokofg** [comment]
https://github.com/zif002/posex/ this repo fixes the problem

**Comment by selwynluo** [comment]
> You can also roll back posex to 0.4. You can do this as described [here](https://github.com/hnmr293/posex/issues/40#issuecomment-1574551072) or you can temporarily download the version I already rolled back https://github.com/daswer123/posex via install from url
> 
> This will not limit you to the gradio version or the extension version.

Works! Great! Thanks!
